packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -176,9 +176,17 @@ export class MarketOperationUtils {
             ? this._sampler.getBalancerSellQuotesOffChainAsync(makerToken, takerToken, sampleAmounts)
             : Promise.resolve([]);
 
-        const offChainBancorPromise = _opts.excludedSources.includes(ERC20BridgeSource.Bancor)
-            ? Promise.resolve([])
-            : this._sampler.getBancorSellQuotesOffChainAsync(makerToken, takerToken, [takerAmount]);
+        const bntToken =
+            this._sampler.bancorService &&
+            (await this._sampler.bancorService.getBancorTokenAddressAsync()).toLowerCase();
+        const shouldSampleBancor =
+            !_opts.excludedSources.includes(ERC20BridgeSource.Bancor) &&
+            bntToken !== undefined &&
+            (makerToken === bntToken || takerToken === bntToken);
```

Comment:
I think we can actually do this in 0xAPI. We know if its Bancor token and can remove Bancor from the excluded lists. By default keep it in the excluded sources.

Reply:
Yeah, looks like we hardcode it in 0x API anyway, so I'll remove refs to token address here.